WRITING.md

```diff
@@ -110,47 +107,56 @@ By default, the badge uses a muted colour scheme to blend in. It also has an acc
 
 ### Since Component
 
-As features are added to Astro, it can be helpful to document _when_ they were added. This allows users to easily see if the version of Astro they are running supports a specific feature as described in the docs.
+As features are added to Astro, it can be helpful to document _when_ they were added. This allows users to easily see if the version of Astro (or other packages) they are running supports a specific feature as described in the docs.
 
-You can use the `<Since />` component to display this information in a standardized way. This component takes a single `v` prop, which indicates the version of Astro in which the feature was added.
+You can use the `<Since />` component to display this information in a standardized way.
 
-```md
----
-setup: |
-  import Since from '~/components/Since.astro';
----
+This component takes two props:
+
+- A `v` prop, which indicates the version of the package in which the feature was added.
+- A `pkg` prop, which indicates which package is being documented. This is optional and will default to `'astro'` so is only required when using `<Since />` for other packages.
+
+```mdx
+import Since from '~/components/Since.astro';
 
 <Since v="1.0.0" />
 ```
 
-This will render the text “**Added in:** v1.0.0”.
+This will render the text “**Added in:** `astro@1.0.0`”.
 
 The advantages of using the component include:
 
 - “Added in” is automatically translated on pages in other languages.
-- The passed version is checked against the current Astro version and a “NEW” badge will be added automatically as long as the version is relatively recent.
+- The passed version is checked against the current Astro version and “NEW” or “BETA” badges will be added automatically based on data from npm.
```

Comment:
Missed this mention of Astro where it could now be any package:

```suggestion
- The passed version is checked against the current package version and “NEW” or “BETA” badges will be added automatically based on data from npm.
```